app/src/main/java/com/keylesspalace/tusky/adapter/FollowRequestViewHolder.kt

```diff
@@ -92,9 +96,11 @@ class FollowRequestViewHolder(
         if (account.note.isEmpty()) {
             binding.accountNote.hide()
         } else {
-            binding.accountNote.text =
-                account.note.parseAsMastodonHtml()
-                    .emojify(account.emojis, binding.accountNote, animateEmojis)
+            binding.accountNote.show()
+
+            val emojifiedNote = account.note.parseAsMastodonHtml()
+                .emojify(account.emojis, binding.accountNote, animateEmojis)
+            setClickableText(binding.accountNote, emojifiedNote, emptyList(), null, linkListener)
```

Comment:
Good catch.

Please can you also change the `TextView` that contains the note in the layout file to `ClickableSpanTextView`, so that the touch targets are expanded appropriately.

If you can't do that today I'll make a change to the PR directly, so that this can go in to beta 3.